app/services/stage_globus.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+# Moves files from the ActiveStorage to the staging mount
+class StageGlobus
+  def self.stage(globus_ids, druid)
+    # Skip side effects if no signed IDs provided
+    return yield if globus_ids.empty?
```

Comment:
I'm wondering whether we would expect this method to yield if in fact it's not doing anything else. For instance, with:

```ruby
[].each do |member|
  Workflow.do_a_thing!
end
```

Would we expect `Workflow.do_a_thing!` to be executed? I don't *think* I would? 🤔